+ source ../common_lib/fsio_lib.sh
++ source /root/mugen/libs/locallibs/common_lib.sh
+++ python3 --version
+++ '[' 0 -eq 0 ']'
+++ source /root/mugen/libs/locallibs/common_lib_python.sh
++ export 'FS_TYPE=ext3 ext4 xfs'
++ FS_TYPE='ext3 ext4 xfs'
++ export LANG=en_US.UTF-8
++ LANG=en_US.UTF-8
++ REMOVE_VG
++ vggroup=
++ [[ '' != '' ]]
+++ df -T
+++ grep test_vggroup
+++ awk '{print $7}'
++ mounted=
++ [[ '' != '' ]]
++ lsblk
++ grep test_vggroup
++ [[ 0 -eq 0 ]]
+++ lsblk
+++ grep test_vggroup
+++ awk '{print $1}'
+++ head -n 1
+++ cut -d - -f 1
++ vggroup=└─test_vggroup20231019225139
++ vggroup=test_vggroup20231019225139
++ vgremove test_vggroup20231019225139 -y
  Logical volume "test_lv20231019225139" successfully removed.
  Volume group "test_vggroup20231019225139" successfully removed
++ rm -rf '/tmp/point*'
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start to prepare the database config.'
+ message='Start to prepare the database config.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to prepare the database config.'
Thu Oct 19 22:51:55 2023 - INFO  - Start to prepare the database config.
++ date +%Y%m%d%H%M%S
+ cur_date=20231019225155
++ CREATE_VG
+++ date +%Y%m%d%H%M%S
++ cur_date=20231019225155
+++ lsblk
+++ grep disk
+++ awk '{print $1}'
+++ tail -n 1
++ free_disk=vdb
++ disk_name=/dev/vdb
++ pvcreate /dev/vdb -f
++ [[ 0 -eq 0 ]]
++ vggroup=test_vggroup20231019225155
++ vgcreate test_vggroup20231019225155 /dev/vdb
++ printf test_vggroup20231019225155
+ vggroup=test_vggroup20231019225155
+ lv=test_lv20231019225155
+ lvcreate -n test_lv20231019225155 -L 2G test_vggroup20231019225155 -y
+ mkfs.ext4 /dev/test_vggroup20231019225155/test_lv20231019225155
mke2fs 1.47.0 (5-Feb-2023)
Discarding device blocks:      0/524288             done                            
Creating filesystem with 524288 4k blocks and 131072 inodes
Filesystem UUID: 0df4a0f9-94f3-4a6d-94fd-aadc85c0b0ce
Superblock backups stored on blocks: 
	32768, 98304, 163840, 229376, 294912

Allocating group tables:  0/16     done                            
Writing inode tables:  0/16     done                            
Creating journal (16384 blocks): done
Writing superblocks and filesystem accounting information:  0/16     done

+ ext4_mount=/mnt/ext4_mount20231019225155
+ mkdir /mnt/ext4_mount20231019225155
+ LOG_INFO 'Finish to prepare the database config.'
+ message='Finish to prepare the database config.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Finish to prepare the database config.'
Thu Oct 19 22:51:57 2023 - INFO  - Finish to prepare the database config.
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start to run test.'
+ message='Start to run test.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to run test.'
Thu Oct 19 22:51:57 2023 - INFO  - Start to run test.
+ mount -t ext4 /dev/test_vggroup20231019225155/test_lv20231019225155 /mnt/ext4_mount20231019225155
+ CHECK_RESULT 0 0 0 'mount ext4 failed.'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='mount ext4 failed.'
+ exit_mode=0
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ df -h
+ grep /mnt/ext4_mount20231019225155
/dev/mapper/test_vggroup20231019225155-test_lv20231019225155  2.0G   24K  1.8G   1% /mnt/ext4_mount20231019225155
+ CHECK_RESULT 0 0 0 'Check mount infos failed.'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='Check mount infos failed.'
+ exit_mode=0
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ umount /mnt/ext4_mount20231019225155
+ CHECK_RESULT 0 0 0 'umount ext4 failed.'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='umount ext4 failed.'
+ exit_mode=0
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
++ blkid
++ grep /dev/mapper/test_vggroup20231019225155-test_lv20231019225155
++ awk '{print $2}'
++ cut -d '"' -f 2
+ uuid=0df4a0f9-94f3-4a6d-94fd-aadc85c0b0ce
+ mount UUID=0df4a0f9-94f3-4a6d-94fd-aadc85c0b0ce /mnt/ext4_mount20231019225155
+ df -h
+ grep /mnt/ext4_mount20231019225155
/dev/mapper/test_vggroup20231019225155-test_lv20231019225155  2.0G   24K  1.8G   1% /mnt/ext4_mount20231019225155
+ CHECK_RESULT 0 0 0 'Check mount infos failed.'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='Check mount infos failed.'
+ exit_mode=0
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ umount /mnt/ext4_mount20231019225155
+ CHECK_RESULT 0 0 0 'umount ext4 failed.'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='umount ext4 failed.'
+ exit_mode=0
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ LOG_INFO 'End to run test.'
+ message='End to run test.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to run test.'
Thu Oct 19 22:51:58 2023 - INFO  - End to run test.
+ CASE_RESULT 0
+ case_re=0
+ test -z ''
+ test 0 -eq 0
+ LOG_INFO 'succeed to execute the case.'
+ message='succeed to execute the case.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'succeed to execute the case.'
Thu Oct 19 22:51:59 2023 - INFO  - succeed to execute the case.
+ exec_result=
+ exit 0
+ post_test
+ LOG_INFO 'Start to restore the test environment.'
+ message='Start to restore the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to restore the test environment.'
Thu Oct 19 22:51:59 2023 - INFO  - Start to restore the test environment.
+ rm -rf /mnt/ext4_mount20231019225155
+ LOG_INFO 'End to restore the test environment.'
+ message='End to restore the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to restore the test environment.'
Thu Oct 19 22:52:00 2023 - INFO  - End to restore the test environment.
